internal/manifests/collector/parser/receiver/receiver_aws-xray.go

```diff
@@ -25,11 +26,12 @@ const parserNameAWSXRAY = "__awsxray"
 // NewAWSXrayReceiverParser builds a new parser for AWS xray receivers, from the contrib repository.
 func NewAWSXrayReceiverParser(logger logr.Logger, name string, config map[interface{}]interface{}) parser.ComponentPortParser {
```

Comment:
this is unused now and will be deleted soon, you can safely remove this

Reply:
Ahh, that explains a lot. I've removed the change, thanks for pointing it out!